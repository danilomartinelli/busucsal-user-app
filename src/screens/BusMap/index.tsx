import * as React from "react";
import MapView from "react-native-maps";

export interface IProps {}

export interface IState {}

export default class BusMap extends React.Component<IProps, IState> {
  constructor(props: IProps) {
    super(props);
    this.state = {};
  }

  public render() {
    return (
      <MapView
        initialRegion={{
          latitude: 37.78825,
          longitude: -122.4324,
          latitudeDelta: 0.0922,
          longitudeDelta: 0.0421,
        }}
      />
    );
  }
}
