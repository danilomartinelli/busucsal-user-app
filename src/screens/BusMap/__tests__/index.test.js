import React from "react";
import { shallow } from "enzyme";

import BusMap from "..";

describe("BusMap Screen", () => {
  describe("render", () => {
    it("should render properly", () => {
      const wrapper = shallow(<BusMap />);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
