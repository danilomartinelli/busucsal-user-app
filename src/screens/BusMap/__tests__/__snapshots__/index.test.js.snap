// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BusMap Screen render should render properly 1`] = `ShallowWrapper {}`;
